runners/jet-experimental/build.gradle

```diff
@@ -42,38 +44,64 @@ dependencies {
     shadowTest library.java.junit
     shadowTest "com.hazelcast.jet:hazelcast-jet-core:$jet_version:tests"
     shadowTest "com.hazelcast:hazelcast:$hazelcast_version:tests"
     shadowTest "com.hazelcast:hazelcast-client:$hazelcast_version:tests"
 
+    needsRunner project(path: ":sdks:java:core", configuration: "shadowTest")
+    needsRunner project(path: ":runners:core-java", configuration: "shadowTest")
+    needsRunner project(path: ":runners:core-construction-java", configuration: "shadowTest")
+    needsRunner project(path: project.path, configuration: "shadowTest")
+
     validatesRunner project(path: ":sdks:java:core", configuration: "shadowTest")
     validatesRunner project(path: ":runners:core-java", configuration: "shadowTest")
+    validatesRunner project(path: ":runners:core-construction-java", configuration: "shadowTest")
     validatesRunner project(path: project.path, configuration: "shadowTest")
 }
 
 task validatesRunnerBatch(type: Test) {
     group = "Verification"
-    systemProperty "beamTestPipelineOptions", JsonOutput.toJson([
-            "--runner=TestJetRunner",
-            "--jetGroupName=jet",
-            "--jetLocalParallelism=2"
-    ])
+    systemProperty "beamTestPipelineOptions", JsonOutput.toJson(["--runner=TestJetRunner"])
 
     classpath = configurations.validatesRunner
     testClassesDirs = files(project(":sdks:java:core").sourceSets.test.output.classesDirs)
     useJUnit {
         includeCategories 'org.apache.beam.sdk.testing.ValidatesRunner'
+        excludeCategories "org.apache.beam.sdk.testing.LargeKeys\$Above100MB"
         excludeCategories 'org.apache.beam.sdk.testing.UsesImpulse' //impulse doesn't cooperate properly with Jet when multiple cluster members are used
         exclude '**/SplittableDoFnTest.class' //Splittable DoFn functionality not yet in the runner
+        exclude '**/ParDoLifecycleTest.class' //some of these tests aren't playing well with our test runner's parallelism...
     }
 
     maxHeapSize = '4g'
 }
 
 task validatesRunner {
     group = "Verification"
-    description "Validates Jet runner"
+    description "Validates Jet Runner"
     dependsOn validatesRunnerBatch
 }
+
+task needsRunnerTests(type: Test) {
+    group = "Verification"
+    description = "Runs tests that require a runner to validate that piplines/transforms work correctly"
+    systemProperty "beamTestPipelineOptions", JsonOutput.toJson(["--runner=TestJetRunner"])
+
+    classpath = configurations.needsRunner
+    testClassesDirs += files(project(":runners:core-construction-java").sourceSets.test.output.classesDirs)
+    testClassesDirs += files(project(":runners:core-java").sourceSets.test.output.classesDirs)
+    testClassesDirs += files(project(":sdks:java:core").sourceSets.test.output.classesDirs)
+    useJUnit {
+        includeCategories "org.apache.beam.sdk.testing.NeedsRunner"
+        excludeCategories "org.apache.beam.sdk.testing.LargeKeys\$Above100MB"
```

Comment:
Are all of these passing?

Reply:
Well, so-so: `697 tests completed, 69 failed, 6 skipped`